app/managers.py

```diff
@@ -0,0 +1,40 @@
+import sqlite3
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema_db.db3")
+        self.table_name = "actors"
+
+    def all(self) -> list:
+        actors_cursor = self._connection.execute(
+            f"SELECT * FROM {self.table_name}"
+        )
+        return actors_cursor.fetchall()
```

Comment:
Sorry, i missed something. This function should return a list of `Actor` instances, not just a list of values.
> - `all` - a method that returns a list of `Actor` instances from DB